### PR TITLE
Add severity to JSON feed

### PR DESCRIPTION
**Pull request by srmish-jfrog**

### CODE REVIEW

**Review by RafaelGSS — COMMENTED** [review]
Can you also update __mocks__ core 1.json?

File: vuln/core/53.json [hunk 1]
```diff
@@ -5,9 +5,12 @@
   "vulnerable": "10.x",
   "patched": ">= 10.9.0",
   "publish_date": "2018-08-16",
-  "author": "Сковорода Никита Андреевич (Nikita Skovoroda / @ChALkeR)",
+  "author": "\u0421\u043a\u043e\u0432\u043e\u0440\u043e\u0434\u0430 \u041d\u0438\u043a\u0438\u0442\u0430 \u0410\u043d\u0434\u0440\u0435\u0435\u0432\u0438\u0447 (Nikita Skovoroda / @ChALkeR)",
```
Comment (RafaelGSS) [inline vuln/core/53.json:8]: Can you revert that?

**Review by RafaelGSS — APPROVED** [review]
Just one comment. LGTM

File: vuln/core/53.json (hunk 1 above)
Comment (srmish-jfrog) [reply vuln/core/53.json]: Sure, fixed

**Pull request merged**